src/App.tsx

```diff
@@ -1,26 +1,222 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable */
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { USER_ID, createTodos, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
 
 export const App: React.FC = () => {
+  const [error, setError] = useState<string | null>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isTitle, setTitle] = useState('');
+  const [loading, setLoading] = useState(false);
```

Comment:
you can get rid of this state
`const isLoading = !!loadingTodoIds.length;`

![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/86923089/ef9b783f-76d3-4c1c-993e-ca0a3b09ccb5)